Shell/NewCNF.cpp

```diff
@@ -1086,12 +1086,9 @@ void NewCNF::processBoolterm(TermList ts, Occurrences &occurrences)
     case Term::SF_ITE: {
       Formula* condition = sd->getCondition();
 
```

Comment:
Analyser is apparently confused by this for-loop in particular. As it's a bit pointless anyway it can go.